pytorch_lightning/tuner/tuning.py

```diff
@@ -51,13 +51,13 @@ def _tune(
             lr_find_kwargs.setdefault('update_attr', True)
             result['lr_find'] = lr_find(self.trainer, model, **lr_find_kwargs)
 
-        self.trainer.state = TrainerState.FINISHED
+        self.trainer.state.status = TrainerStatus.FINISHED
 
         return result
 
     def _run(self, *args: Any, **kwargs: Any) -> None:
         """`_run` wrapper to set the proper state during tuning, as this can be called multiple times"""
-        self.trainer.state = TrainerState.TUNING  # last `_run` call might have set it to `FINISHED`
+        self.trainer.state.status = TrainerStatus.RUNNING  # last `_run` call might have set it to `FINISHED`
```

Comment:
is this part of the fix you mentioned?

Reply:
No tuning fix in this PR. Those were in previous PRs.